src/components/User/User.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './User.scss';
+import classNames from 'classnames';
+
+export const User = ({ user }) => (
+  <>
+    <strong className={
+      classNames('userName',
+        { userName__done: user.completed })}
+    >
```

Comment:
Ось це здається не до кінця зрозуміло розділено...
```suggestion
    <strong className={ classNames(
        'userName',
        { userName__done: user.completed }
      )}
    >
```
Незн))